src/components/VideoCards/VideoCards.jsx

```diff
@@ -1,9 +1,20 @@
+import { useState } from 'react';
+
 import useStore from '../../hooks/useStore';
+import DeleteModal from '../DeleteModal/DeleteModal';
+import Icon from '../Icon';
+import StyledButton from '../StyledButton';
 import StyledVideoContainer from '../StyledVideoContainer';
 import StyledVideoFrame from '../StyledVideoFrame';
 import StyledVideoTitle from '../StyledVideoTitle';
 
 export default function VideoCard() {
+	const [isShown, setIsShown] = useState(false);
+
+	function handleDelete() {
+		setIsShown(!isShown);
+	}
```

Comment:
`handleDelete` is not a good name for this function. When reading this, I would think that it is a function to handle your delete functionality. What it actually does is to toggle the modal. I would recommend to change the function name.

Reply:
Thanks, I've changed it to ```handleVisibility```